tests/test_gallery.py

```diff
@@ -289,6 +290,31 @@ def test_gallery(settings, tmpdir):
         logger.setLevel(logging.INFO)
 
 
+def test_gallery_max_img_pixels(settings, tmpdir, monkeypatch):
+    "Test the Gallery class with the max_img_pixels setting."
+    monkeypatch.setattr('PIL.Image.MAX_IMAGE_PIXELS', 100000000)
+
+    with open(str(tmpdir.join('my.css')), mode='w') as f:
+        f.write('color: red')
+
+    settings['destination'] = str(tmpdir)
+    settings['user_css'] = str(tmpdir.join('my.css'))
+    settings['max_img_pixels'] = 5000
+
+    logger = logging.getLogger('sigal')
+    logger.setLevel(logging.DEBUG)
+    try:
+        with pytest.raises(PILImage.DecompressionBombError):
+            gal = Gallery(settings, ncpu=1)
+            gal.build()
+
+        settings['max_img_pixels'] = 100000000
```

Comment:
`monkeypatch` sets `MAX_IMAGE_PIXELS` to the same value above, why ?

Reply:
Can you also use `100_000_000` instead for better readability ?

Reply:
Sure, added now.

Reply:
> `monkeypatch` sets `MAX_IMAGE_PIXELS` to the same value above, why ?

I'm using `monkeypatch` to make sure the value goes "back to default" after the test.  The value here doesn't really matter, since I am setting it explicitly in the test.

Reply:
Oh I missed the 5000 one above. Can you add a comment explaining why `monkeypatch` is used ?

Reply:
OK, done.